File: android/app/src/main/java/com/golfiq/hud/model/Telemetry.kt
package com.golfiq.hud.model

data class Telemetry(
    val timestampMs: Long,
    val club: String?,
    val ballSpeed: Double?,
    val clubSpeed: Double?,
    val launchAngle: Double?,
    val spinRpm: Int?,
    val carryMeters: Double?
)

File: ios/Models/Telemetry.swift
import Foundation

public struct Telemetry: Codable {
    public let timestampMs: Int64
    public let club: String?
    public let ballSpeed: Double?
    public let clubSpeed: Double?
    public let launchAngle: Double?
    public let spinRpm: Int?
    public let carryMeters: Double?
}
